tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""Test suite for tap-postgres."""
+
+SQLALCHEMY_URL = "postgresql://postgres:postgres@localhost:5432/postgres"
```

Comment:
While this works, can we keep this in just `tests/test_core.py` so we have one place to look?

Reply:
Yah!

Reply:
Wait do you want this stored separately at the top of each file, or you want it in test_core and then import it into test_replication_key from test_core?